### ISSUE TITLE
Add support for adding peers by IP:port

### ISSUE THREAD

**Issue by Avamander** [jfurrow/flood#640]
## Note

- [X] You may try to follow the update procedure described in the README and try again before opening this issue.
- [X] Before asking for something, please check the [F.A.Q.](https://github.com/jfurrow/flood/wiki/F.A.Q.).
- [X Before opening a bug issue, please check the [Troubleshooting](https://github.com/jfurrow/flood/wiki/Troubleshooting) wiki section.
- [X] If you want to contribute to the project please review the [contributing guidelines](https://github.com/jfurrow/flood/blob/master/.github/CONTRIBUTING.md).
- [X] Keep in mind that Flood is a [FLOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software#FLOSS) (Free, Libre and Open Source Software), so please try to provide a PR when opening a bug issue. Without contributions the project can't live and with your help fix and request will come faster.
- [X] The project is accepting issues (bugs report), feature or enhancement requests, discussions and questions but **not personal support**.

## Summary
Would it be possible to add a small "+" to "Torrent details"->"Peers" that allows adding a new peer by IP:port? As far as I understand rtorrent supports it via `d.add_peer=host[:port]` but it would be nice to have access to the feature via UI.

**Comment by noraj** [comment]
Hi @Avamander 
Have you a use case for that? I can't image a way you need to manually add peers right now because if a peer is available the tracker will know it and your bittorrent client will add it to the list of connected peers if it match to your peers limit policy.

**Comment by Avamander** [comment]
> Have you a use case for that?

Yes. For example when no trackers can be contacted (or should be contacted) and when I know there's a specific host I can get the files from. Say two PCs on the same network, one has the whole file, other doesn't, adding the LAN IP would allow downloading from that first PC.

**Comment by SanPilot** [comment]
This shouldn't be too difficult. rtorrent supports `d.add_peer=host[:port]` syntax since 0.8.4, so we can add button/ui in the peers tab of torrent details and a corresponding API endpoint in the backend.
